groovy/postInit/chemistry/inorganic_chemistry/elements/d_block/group10/PlatinumGroupChain.groovy

```diff
@@ -295,7 +295,7 @@ VACUUM_CHAMBER = recipemap('vacuum_chamber')
         CENTRIFUGE.recipeBuilder()
             .fluidInputs(fluid('silver_free_pgm_solution') * 1130)
             .fluidInputs(fluid('dibutyl_carbitol') * 10)
-            .fluidOutputs(fluid('gold_extract') * 20)
+            .fluidOutputs(fluid('gold_extract') * 10)
```

Comment:
this change would change Au distribution from 1% to 0.5%
1% should be the expected rate right?
![image](https://github.com/user-attachments/assets/4da429ee-641f-4baa-b009-3c69cdf77c83)